### PR TITLE
REPORTING: Change potentiometer report style

### PR DESCRIPTION
**Pull request by Jeff-Ciesielski**
This makes it regular and parseable in our cutting system

@keyme/robotics 

Tested locally

### CODE REVIEW

File: system.c [hunk 1]
```diff
@@ -379,8 +379,11 @@ uint8_t system_execute_line(char *line)
             case 'R':
             {
               const uint8_t result = ad5121_read_pot((enum AD5121_ID)dev_id);
+	      printPgmString(PSTR("[POT<"));
```
Comment (zsalzbank) [inline system.c]: :smile:

Comment (penguinland) [reply system.c]: To rephrase, don't mix tabs and spaces. There ought to be a way to configure emacs to turn tabs into spaces automatically. It's easy in vim... :-)

Comment (Jeff-Ciesielski) [reply system.c]: This is just a gateway commit.

Comment (Jeff-Ciesielski) [reply system.c]: It's already done in my emacs config, this is just a byproduct of having multiple configurations and forgetting to switch. @penguinland

Comment (Jeff-Ciesielski) [reply system.c]: Also, mixed Tabs and spaces is the one true way.  Heretics.

File: system.c [hunk 2]
```diff
@@ -379,8 +379,11 @@ uint8_t system_execute_line(char *line)
             case 'R':
             {
               const uint8_t result = ad5121_read_pot((enum AD5121_ID)dev_id);
+              printPgmString(PSTR("[POT<"));
+              print_uint8_base10(dev_id);
+              printPgmString(PSTR(">:"));
               print_uint8_base10(result);
-              printPgmString(PSTR("\r\n"));
+              printPgmString(PSTR("]\r\n"));
```
Comment (StephanCoetzee) [inline system.c:386]: Why not use `snprintf` and then only call `printPgmString` once?

Comment (Jeff-Ciesielski) [reply system.c:386]: PrintPgmString uses strings stored in flash.

**Review by StephanCoetzee — APPROVED** [review]
LGTM!

**Pull request merged**